.gitignore

```diff
@@ -18,3 +18,4 @@
 burrow-exporter
 dist/
 vendor/
+.idea/
```

Comment:
I'd prefer to keep IDE specific things out of here. I'd recommend adding `.idea/` to your global gitignore config.